azuread/azuread_test.go

```diff
@@ -35,9 +36,31 @@ func TestAzureSqlAuth(t *testing.T) {
 
 }
 
+func TestTDS8ConnWithAzureSqlAuth(t *testing.T) {
+	mssqlConfig := testConnParams(t, ";encrypt=strict;TrustServerCertificate=false;tlsmin=1.2")

```

Comment:
>encrypt=strict;TrustServerCertificate=false;tlsmin=1. [](http://example.com/codeflow?start=36&length=53)

it's possible the test parameters or .azureconnstr file is using a URL instead of ADO format.
Maybe pass a `func(p *azureFedAuthConfig)` instance instead to set specific properties in the config